apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -18,108 +18,169 @@ export function InstructorCourseAdminSettings({
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="title">Title</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="title"
+                  name="title"
+                  value="${resLocals.course.title}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  This is the official title of the course, as given in the course catalog.
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="institution">Institution</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="institution"
+                  name="institution"
+                  value="${resLocals.institution.short_name} (${resLocals.institution.long_name})"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  This is your academic institution (e.g., "University of Illinois").
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="display_timezone">Timezone</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="display_timezone"
+                  name="display_timezone"
+                  value="${resLocals.course.display_timezone}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  The allowable timezones are from the tz database. It's best to use a city-based
+                  timezone that has the same times as you.
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="path">Path</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="path"
+                  name="path"
+                  value="${resLocals.course.path}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  The path where course files can be found.
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="repository">Repository</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="repository"
+                  name="repository"
+                  value="${resLocals.course.repository}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  The Github repository that can be used to sync course files.
+                </small>
+              </div>
+              <div class="form-group">
```

Comment:
A `<p>` would be better here, I think. `class="mb-0"` will keep consistent spacing within the card.

Reply:
Updated to `<p>` with `class="mb-0"` in [7af52d3](https://github.com/PrairieLearn/PrairieLearn/pull/9386/commits/7af52d32ca07211a57026a795ad42cd8870188cf).